libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -320,7 +320,7 @@ def add_version_header(tc):
             "name": "__cpp_lib_constexpr_algorithms",
             "values": {
                 "c++20": 201806,
-                # "c++26": 202306, # P2562R1 constexpr Stable Sorting
+                "c++26": 202306,  # P2562R1 constexpr Stable Sorting
```

Comment:
I added these comments to make it easier to find the FTM for papers when not implemented.
I don't think they add value now the paper is completed. WDYT?
```suggestion
                "c++26": 202306,
```

Reply:
Agreed. I'll remove the comments.